dev_tools/nyaml2nxdl/nyaml2nxdl_backward_tools.py

```diff
@@ -332,6 +333,53 @@ def clean_and_organise_text(self, text, depth):
 
         return text
 
+    def check_and_handle_doc_xref_and_other_doc(self, text, indent):
```

Comment:
This part should also be tested

Reply:
test part has been written for this part..

Reply:
thanks

Reply:
The test is not working properly. If I change parts of the nxdl it still passes the tests.